pennylane/interfaces/tf.py

```diff
@@ -98,10 +101,20 @@ def __repr__(self):
         metric_tensor = qnode.metric_tensor
         draw = qnode.draw
         func = qnode.func
+        num_variables = property(lambda self: qnode.num_variables)
+        arg_vars = property(lambda self: qnode.arg_vars)
 
     @qnode_str
     @tf.custom_gradient
     def _TFQNode(*input_, **input_kwargs):
+        # determine which input tensors are being recorded for backpropagation
+        requires_grad = [
+            tape_lib.should_record_backprop([1 * i])
```

Comment:
What is the return type of this method? Boolean?

Reply:
Yep! Will expand my code comment to include this.